react.js

```diff
@@ -50,6 +50,11 @@ module.exports = {
     "react/jsx-no-duplicate-props": "error",                  // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-duplicate-props.md
     "react/jsx-no-undef": "error",                            // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-undef.md
     "react/jsx-pascal-case": "error",                         // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-pascal-case.md
+    "react/jsx-sort-props": ["error", {                       // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-sort-props.md
+      "callbacksLast": true,
```

Comment:
Why this one?

Reply:
I tend to think of the callbacks like `onClick`, `onDrag` as a different kind of prop from the rest. I'm fine removing this though.

Reply:
@dkniffin I tend to agree with you on this. I think of `onClick`, `onChange`, etc. as somewhat different than the rest. If we're going to separate reserved keys from normal ones it makes sense in my mind to have the callback ones separated as well